src/Uno.UI/Microsoft/UI/Xaml/Controls/TreeView/TreeView.cs

```diff
@@ -258,10 +263,27 @@ private void OnPropertyChanged(DependencyPropertyChangedEventArgs args)
 		{
 			var items = SelectedItems;
 			var selected = items?.Count > 0 ? items[0] : null;
+#if !HAS_UNO
```

Comment:
I'm assuming we have to workaround the issue with Uno-specific code due to a bug somewhere else. In case you found exactly what the actual bug is, it would be great to document it.